website/source/guides/cluster/autopilot.html.md

```diff
@@ -193,11 +199,11 @@ to voters and demoting the old servers. After this is finished, the old servers
 safely removed from the cluster.
 
 To check the Nomad version of the servers, either the [autopilot health]
-(/api/operator.html#read-health) endpoint or the `Nomad members`
+(/api/operator.html#read-health) endpoint or the `nomad members`
 command can be used:
 
 ```
-$ Nomad members
+$ nomad members
```

Comment:
These examples should be changed to be `nomad server-members`. The output is slightly different